src/docs/teams/debugger/using-the-debugger.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Teams | Using the Debugger
+layout: docs.hbs
+---
+# Using the Debugger
+
+In this section we'll explore both the various elements of the debugger's user interface and steps invovled in actually debugging a transaction. Below is an example of the Truffle Teams debugger interface, wherein a simple ERC721 token implementation is being debugged. 
+
+<figure class="screenshot">
+  <img class="figure-shadow mb-2 w-100" src="/img/docs/teams/debugger-transaction-01.png" alt="Teams Debugger Interface">
+  <figcaption class="text-center">The Truffle Teams Debugger Interface.</figcaption>
+</figure>
+
+Using the above example as a reference, the first thing to note is the transaction hash (`0x82f668a...`) at the very top of the screen. To the left of this is a visual representation of whether the transaction ultimately succeeded <i class="fas fa-check-circle" style="color: #00A311"></i> or failed <i class="fas fa-times-circle" style="color: #D60000"></i>. This is a useful reference as it allows you to preemptively know whether to expect a status message when execution completes. More on this in the [Status Messages](/docs/teams/debugger/using-the-debugger#status-messages) section below.
+
+## Debugger Control Palette
+
+Located directly below the transaction hash is the debugger's control palette.
+
+<figure class="screenshot">
+  <img class="figure-shadow mb-2 w-50" src="/img/docs/teams/debugger-control-palette.png" alt="Teams Debugger Control Palette">
+  <figcaption class="text-center">The Truffle Teams Debugger Control Palette.</figcaption>
+</figure>
+
+From left to right, the controls, with a description of their associated action, are as follows. Note that you can also hover a control to display its associated name.
+
+- <code>Continue</code> - continue until the next breakpoint is reached or the last line is executed
+- <code>Step Over</code> - steps over the current line
+- <code>Step Into</code> - steps into the function call or contract creation currently being evaluated
+- <code>Step Out</code> - steps out of the currently running function
+- <code>Restart</code> - restarts the debugger session
```

Comment:
Instead of saying "from left to right" and naming the buttons, can you say "below is a description of what each button does...". Then, in addition to your existing names of the buttons, include the button visual representation with the respective icon

Reply:
noted 👍